pkg/controller/controller.go

```diff
@@ -93,6 +93,16 @@ type TypedOptions[request comparable] struct {
 	//
 	// Note: This flag is disabled by default until a future version. It's currently in beta.
 	UsePriorityQueue *bool
+
+	// EnableWarmup specifies whether the controller should start its sources when the manager is not
```

Comment:
Maybe sync this comment with the one in pkg/config/controller.go (as this field is also exported)